artifacts/definitions/Server/Import/CuratedSigma.yaml

```diff
@@ -22,15 +22,19 @@ parameters:
       - Velociraptor Curated Windows Ruleset
 
   - name: Prefix
-    description: Add artifacts with this prefix
+    description: Add this prefix to imported artifacts
+    validating_regex: '^[a-zA-Z0-9_.]$'
 
 sources:
   - query: |
+      LET PackageNames <= parse_json_array(data=PackageNames)
```

Comment:
Is that needed? the compiler should add that in. If not we should add it to compiler.go

Reply:
yes it's currently needed. The multichoice selctor presents a string.

Reply:
Ok lets fix it :-)

Reply:
Very strange. I can't see why it's not matching in compiler.go
![Screenshot from 2024-04-26 14-51-56](https://github.com/Velocidex/velociraptor/assets/9843422/e7f1280b-b8b8-4167-a5c1-cae4b22aca17)

Reply:
I dont think this case is handled at all - we need to add it there

Reply:
Should this not handle it?: https://github.com/Velocidex/velociraptor/blob/c2fd3c4898039c491e894fd236abcdfbd71d5c85/services/launcher/compiler.go#L145

Reply:
Almost - that case statement applies for "json_array", "regex_array" but we need to also apply it to multichoice which uses json array under the hood (but the GUI does something else with it)

Reply:
Thanks, I understand now.